.github/workflows/pr.yml

```diff
@@ -121,6 +121,22 @@ jobs:
           fetch-depth: 0
       - name: Check if dockerimage build is working
         run: docker build -f ./Dockerfile .
+  build-cdq-analysis-image:
+    name: Check CDQ Analysis Image Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Change to the cdq-analysis directory
+        run: cd cdq-analysis/
+      - name: Check if dockerimage build is working
+        run: docker build -f ./Dockerfile .
```

Comment:
We should use the same docker buildx build that the workflow on pushes to main uses, so that the builds on PR checks and post-merge are consistent

Reply:
but the post-merge action is to push the image to dockerhub, and we do not want to push images to dockerhun for PR builds, right?

Reply:
Hmm, didn't noticed the build _and_ push in that step. I think copying over the docker buildx setup step should be sufficient then, and then just keeping the `docker build -f ./Dockerfile`